website/content/docs/internals/telemetry/metrics/storage.mdx

```diff
@@ -0,0 +1,175 @@
+---
+layout: docs
+page_title: "Telemetry reference: Storage plugin metrics"
+description: >-
+  Technical reference for individual storage plugin telemetry values.
+---
+
+# Storage plugin telemetry
```

Comment:
Might want to link to the raft page for those using integrated storage.

Reply:
done